packages/twenty-front/src/modules/settings/components/SettingsOptionCardContent.tsx

```diff
@@ -52,44 +94,99 @@ const StyledToggle = styled(Toggle)`
   margin-left: auto;
 `;
 
+const StyledSelect = styled(Select)`
+  margin-left: auto;
+  width: 120px;
+`;
+
 const StyledCover = styled.span`
   cursor: pointer;
   inset: 0;
   position: absolute;
 `;
 
-export const SettingsOptionCardContent = ({
-  Icon,
-  title,
-  description,
-  divider,
-  checked,
-  onChange,
-}: SettingsOptionCardContentProps) => {
+export const SettingsOptionCardContent = (
+  props: SettingsOptionCardContentProps,
+) => {
   const theme = useTheme();
-
   const toggleId = useId();
+  const {
+    Icon,
+    title,
+    description,
+    divider,
+    disabled = false,
+    advancedMode = false,
+  } = props;
+
+  const renderControl = () => {
```

Comment:
this will also remove this (we don't render components through render() function in the project, we prefer split). I'm not sure this is great for performance reasons also